doc/index.rst

```diff
@@ -25,10 +25,10 @@ indicators.
 Try out
 =======
 
-Flycheck needs GNU Emacs |min-emacs| or newer, and works best on Unix systems.
-**Windows users**, please be aware that Flycheck does not support Windows
-officially, although it should mostly work fine on Windows.  See :ref:`Windows
-support <flycheck-windows-support>` and watch out for `known Windows issues`_!
+Flycheck needs GNU Emacs |min-emacs|, and works best on Unix systems.  **Windows
```

Comment:
reST tip: you can escape spaces with a backslash followed by a space: `|min-emacs|\ +`